partner_pricelist_search/models/product_pricelist.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class Pricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    def show_pricelist_partners(self):
+        if len(self) == 1:
+            domain = [("property_product_pricelist", "=", self.id)]
+        else:
+            domain = [("property_product_pricelist", "in", self.ids)]
+        partners = self.env["res.partner"].search(domain)
+        action = self.env.ref("base.action_partner_form")
+        res = action.read()[0]
+        res["domain"] = [
+            ("id", "in", partners.ids),
+        ]
+        return res
```

Comment:
I think it worth
```suggestion
    def show_pricelist_partners(self):
        action = self.env.ref("base.action_partner_form").read()[0]
        if len(self) == 1:
            res["domain"] =  [("property_product_pricelist", "=", self.id)]
        else:
            res["domain"] = [("property_product_pricelist", "in", self.ids)]
        return action
```

Reply:
Sorry, I didn't see the suggestion.
Right now I don't remember why I did like this.
I will try that in the next PR or if you prefer you can do your PR and I approve it